.github/ISSUE_TEMPLATE/team_issues.yaml

```diff
@@ -16,8 +16,8 @@ body:
       required: true
   - type: textarea
     attributes:
-      label: Suggestion
-      description: Do you have a suggestion to close the issue?
+      label: Why
```

Comment:
```suggestion
      label: How?
```

Reply:
Oops 🤦

Reply:
Fixed in https://github.com/grafana/xk6-browser/pull/1024/commits/bca88d498c27e63022e0fa243f5230673a802ff5.